app/routers/user_router.py

```diff
@@ -4,16 +4,28 @@
 from app.utils.token import create_access_token
 from bson import ObjectId
 
+# Suppress UserWarning from pydantic
+warnings.filterwarnings("ignore", category=UserWarning, module="pydantic")
```

Comment:
What is the purpose of this, asking for knowledge more than anything.

Reply:
When creating a user, related to the way we handle the ids, there was always a warning related to pydantic.